src/runtime/auto-prompt-manager.ts

```diff
@@ -746,9 +733,28 @@ export class AutoPromptManager {
           continue;
         }
       }
-      return action;
+      potentialAction = action;
+      break;
+    }
+
+    if (!potentialAction) {
+      return;
+    }
+
```

Comment:
Can you add some comments to make it clear that the order between prompt/global frequency cap evaluation matters and this is the intended order?

Reply:
Done.